.github/workflows/build.yml

```diff
@@ -21,86 +21,59 @@ jobs:
       with:
         submodules: recursive
 
-    - name: Cache vcpkg binaries
-      uses: actions/cache@v3
-      with:
-        path: ${{ runner.os == 'Windows' && 'C:\\Users\\runneradmin\\AppData\\Local\\vcpkg\\' || '~/.cache/vcpkg' }}
-        key: vcpkg-binary-${{ matrix.os }}-${{ hashFiles('vcpkg.json') }}
-        restore-keys: |
-          vcpkg-binary-${{ matrix.os }}-
-    - name: Cache vcpkg directory
-      uses: actions/cache@v3
-      with:
-        path: vcpkg
-        key: vcpkg-${{ matrix.os }}-${{ hashFiles('vcpkg.json') }}
-        restore-keys: |
-          vcpkg-${{ matrix.os }}-
-    - name: Set up vcpkg for Windows
-      if: runner.os == 'Windows'
-      shell: pwsh
-      run: |
-        if (Test-Path -Path "vcpkg") {
-          Remove-Item -Recurse -Force "vcpkg"
-        }
-        git clone https://github.com/microsoft/vcpkg.git
-        cd vcpkg
-        .\bootstrap-vcpkg.bat
-        .\vcpkg integrate install
-    - name: Set up vcpkg for macOS/Linux
-      if: runner.os != 'Windows'
-      run: |
-        rm -rf vcpkg
-        git clone https://github.com/microsoft/vcpkg.git
-        cd vcpkg
-        ./bootstrap-vcpkg.sh
-        ./vcpkg integrate install
     - name: Set up dependencies for macOS and Linux
       if: runner.os != 'Windows'
       run: |
         if [ "${{ runner.os }}" == "macOS" ]; then
-          brew install cmake gfortran curl zip unzip gnu-tar
+          brew install cmake ninja
```

Comment:
this step could be covered by the [get-cmake](https://github.com/marketplace/actions/get-cmake) action, but it's not a big enough deal to change it now